research/10.md

```diff
@@ -0,0 +1,20 @@
+# Label: 10
+
+## Description
+
+Used by Southwest (WN)
```

Comment:
I would imagine they aren't the only airline that uses this label, perhaps a note about that it's the currently only known one, just in case it might prevent someone from contributing something contrary.

Reply:
Yes, I can change it.